qabelbox/src/main/java/de/qabel/qabelbox/settings/fragments/SettingsFragment.kt

```diff
@@ -0,0 +1,136 @@
+package de.qabel.qabelbox.settings.fragments
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.DialogInterface
+import android.content.Intent
+import android.content.IntentFilter
+import android.os.Bundle
+import android.preference.CheckBoxPreference
+import android.preference.Preference
+import android.preference.PreferenceFragment
+
+import org.apache.commons.io.FileUtils
+
+import javax.inject.Inject
+
+import de.qabel.qabelbox.QblBroadcastConstants
+import de.qabel.qabelbox.R
+import de.qabel.qabelbox.account.AccountManager
+import de.qabel.qabelbox.config.AppPreference
+import de.qabel.qabelbox.helper.UIHelper
+import de.qabel.qabelbox.index.ContactSyncAdapter
+import de.qabel.qabelbox.index.preferences.AndroidIndexPreferences
+import de.qabel.qabelbox.index.preferences.IndexPreferences
+import de.qabel.qabelbox.settings.SettingsActivity
+import de.qabel.qabelbox.settings.navigation.SettingsNavigator
+import de.qabel.qabelbox.storage.model.BoxQuota
+
+class SettingsFragment : PreferenceFragment() {
+
+    companion object {
+        val APP_PREF_NAME = AndroidIndexPreferences.KEY
```

Comment:
`SettingsFragment.companion.getAPP_PREF_NAME` is maximum ugly ^^
There is an annotation that allows to drop the `.companion` and I think if you make APP_PREF_NAME a `const val ...`, you can access it as a field from Java. If not, please add a getter for the CrashReportingActivity or convert it to kotlin.